### PR TITLE
chore: align with frontend library template

### PR DESCRIPTION
**Pull request by artursvonda**
Replaced release process with the new one, ran linter on files.

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 4,
+  "singleQuote": true,
+  "trailingComma": "all"
```
Comment (artursvonda) [inline .prettierrc.json:4]: This just aligns with what we currently had and I'd argue that `singleQuote` and `trailingComma` is something we should have part of our template.

Also, should we make `tab` the default?

File: package.json [hunk 2]
```diff
@@ -26,8 +26,10 @@
   ],
   "scripts": {
     "test": "jest",
-    "prepare-release": "standard-version",
-    "prepare": "husky install"
+    "lint": "eslint . --ext .js,.cjs,.ts",
```
Comment (rupert-mckay) [inline package.json:29]: 🟢 I didn't really matter, but you don't need `.ts` here.

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "settings": {
+    "jest": {
+      "version": ">= 27"
```
Comment (rupert-mckay) [inline .eslintrc.json:4]: 🔶 This is only necessary when using something other than `jest` and we need to tell the Jest lint rules what version of Jest's syntax we are using.

(e.g. vitest in the template project)

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    }
+  },
+  "extends": [
+    "@lokalise/lokalise-frontend/core",
+    "@lokalise/lokalise-frontend/jest"
+  ],
+  "overrides": [
```
Comment (rupert-mckay) [inline .eslintrc.json:11]: 🔶 This override is no longer necessary with eslint-config v0.3.1

**Review by rupert-mckay — CHANGES_REQUESTED** [review]
I think you should remove the `settings` and `overrides` parts of the eslintrc. See comments in file for the reason.

**Review by rupert-mckay — APPROVED** [review]

File: .prettierrc.json (hunk 1 above)
Comment (sangdth) [reply .prettierrc.json:4]: For accessibility, I support the `tab` by default.

**Pull request merged**